### ISSUE TITLE
Decks.confForDid crash

### ISSUE THREAD

**Issue by mikehardy** [ankidroid/Anki-Android#6880]
###### Reproduction Steps

1. unknown
2. 
3. 

###### Actual Result

```
java.lang.RuntimeException: 
  at android.app.ActivityThread.performLaunchActivity (ActivityThread.java:3430)
  at android.app.ActivityThread.handleLaunchActivity (ActivityThread.java:3614)
  at android.app.servertransaction.LaunchActivityItem.execute (LaunchActivityItem.java:86)
  at android.app.servertransaction.TransactionExecutor.executeCallbacks (TransactionExecutor.java:108)
  at android.app.servertransaction.TransactionExecutor.execute (TransactionExecutor.java:68)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:2199)
  at android.os.Handler.dispatchMessage (Handler.java:112)
  at android.os.Looper.loop (Looper.java:216)
  at android.app.ActivityThread.main (ActivityThread.java:7625)
  at java.lang.reflect.Method.invoke (Method.java)
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:524)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:987)
Caused by: java.lang.NullPointerException: 
  at com.ichi2.libanki.Decks.confForDid (Decks.java:690)
  at com.ichi2.anki.AbstractFlashcardViewer.getConfigForCurrentCard (AbstractFlashcardViewer.java:2284)
  at com.ichi2.anki.AbstractFlashcardViewer.loadContentIntoCard (AbstractFlashcardViewer.java:2322)
  at com.ichi2.anki.AbstractFlashcardViewer.lambda$fillFlashcard$8 (AbstractFlashcardViewer.java:2309)
  at com.ichi2.anki.-$$Lambda$AbstractFlashcardViewer$8BjytBMw6ioa9YknPETK2LJcsts.consume (-.java:6)
  at com.ichi2.anki.AbstractFlashcardViewer.lambda$processCardAction$0 (AbstractFlashcardViewer.java:482)
  at com.ichi2.anki.-$$Lambda$AbstractFlashcardViewer$iGYhHuwUWNEvupUJDD5-611bphY.apply (-.java:4)
  at com.ichi2.anki.AbstractFlashcardViewer.processCardFunction (AbstractFlashcardViewer.java:492)
  at com.ichi2.anki.AbstractFlashcardViewer.processCardAction (AbstractFlashcardViewer.java:481)
  at com.ichi2.anki.AbstractFlashcardViewer.fillFlashcard (AbstractFlashcardViewer.java:2309)
  at com.ichi2.anki.AbstractFlashcardViewer.updateCard (AbstractFlashcardViewer.java:2182)
  at com.ichi2.anki.AbstractFlashcardViewer.displayCardQuestion (AbstractFlashcardViewer.java:1953)
  at com.ichi2.anki.AbstractFlashcardViewer.displayCardQuestion (AbstractFlashcardViewer.java:1919)
  at com.ichi2.anki.CardTemplatePreviewer.displayCardQuestion (CardTemplatePreviewer.java:136)
  at com.ichi2.anki.CardTemplatePreviewer.onCollectionLoaded (CardTemplatePreviewer.java:195)
  at com.ichi2.anki.AnkiActivity.startLoadingCollection (AnkiActivity.java:325)
  at com.ichi2.anki.CardTemplatePreviewer.onCreate (CardTemplatePreviewer.java:86)
  at android.app.Activity.performCreate (Activity.java:7458)
  at android.app.Activity.performCreate (Activity.java:7448)
  at android.app.Instrumentation.callActivityOnCreate (Instrumentation.java:1286)
  at android.app.ActivityThread.performLaunchActivity (ActivityThread.java:3409)
  at android.app.ActivityThread.handleLaunchActivity (ActivityThread.java:3614)
  at android.app.servertransaction.LaunchActivityItem.execute (LaunchActivityItem.java:86)
  at android.app.servertransaction.TransactionExecutor.executeCallbacks (TransactionExecutor.java:108)
  at android.app.servertransaction.TransactionExecutor.execute (TransactionExecutor.java:68)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:2199)
  at android.os.Handler.dispatchMessage (Handler.java:112)
  at android.os.Looper.loop (Looper.java:216)
  at android.app.ActivityThread.main (ActivityThread.java:7625)
  at java.lang.reflect.Method.invoke (Method.java)
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:524)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:987)
```

###### Debug info
Refer to the [support page](https://ankidroid.org/docs/help.html) if you are unsure where to get the "debug info".

###### Research
*Enter an [x] character to confirm the points below:*

- [ ] I have read the [support page](https://ankidroid.org/docs/help.html) and am reporting a bug or enhancement request specific to AnkiDroid
- [ ] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue
- [ ] I have searched for similar existing issues here and on the user forum
- [ ] (Optional) I have confirmed the issue is not resolved in the latest alpha release ([instructions](https://docs.ankidroid.org/manual.html#betaTesting))

**Comment by david-allison-1** [comment]
Should be fixed by a check database - was this a crash, or an ACRA report?

**Comment by mikehardy** [comment]
play console exclusively gets full crashes

**Comment by dae** [comment]
Closed by #14171.
